pkg/executor/simple.go

```diff
@@ -1122,6 +1123,11 @@ func (e *SimpleExec) executeCreateUser(ctx context.Context, s *ast.CreateUserStm
 
 	users := make([]*auth.UserIdentity, 0, len(s.Specs))
 	for _, spec := range s.Specs {
+		if !s.IsCreateRole {
+			if err := keyspace.GetUsernamePolicy().ValidateUsername(spec.User.Username); err != nil {
```

Comment:
If our interface is a generic interface, why do we need to introduce the content under the keyspace package here? Or should this func be moved to another package?